src/fr/pages/index.md

```diff
@@ -5,4 +5,73 @@ permalink: /fr/
 translationKey: "index"
 ---
 
-# Get Started FR
+<header>
+
+BIENVENUE
+
+# Système de design du gouvernement du Canada
+
+</header>
+
+## Liens rapides
+
+<ul>
+    <li>
+        <a href="{{ "/fr/composants/formulaires/aide-a-la-conception/" | url }}">Design de formulaires</a>
+    </li>
+    <li>
+        <a href="https://github.com/cds-snc/gcds-components">Dépôt GitHub</a>
```

Comment:
Can you add an aria-label here that this opens in a new tab, please?
(There is an example with french translation in the header.njk file )